### DISCUSSION TITLE
Smartmeter configuration in config.ini

### DISCUSSION THREAD

**Discussion by z-master42** [reinhard-brandstaedter/solarflow-control#101] [Q&A] [answered]
What do cur_accessor and total_accessor look like if the data after the base topic is not JSON?
The data structure is similar to solarflow-hub.

![grafik](https://github.com/reinhard-brandstaedter/solarflow-control/assets/66380371/ffedfa78-45bc-4aab-a10b-a45488e89dff)

**Comment by reinhard-brandstaedter** [comment]
What type of smartmeter reader is this?
chn2/raw seems to be the current usage?

**Reply by z-master42** [reply]
Correct.
chn0/agg is 1.8.0 in Wh and chn1/agg is 2.8.0 in Wh.

This is a Raspberry Pi with USB infrared reading head. The Pi is running vzlogger from the [volkszaehler ](https://volkszaehler.org/)project.

**Reply by reinhard-brandstaedter** [reply, marked as answer]
You can try the letest :dev build/image.
I’ve added a vzlogger smartmeter type. Please also see the config.ini template and point the topic to the ../raw topic

**Reply by z-master42** [reply]
Very cool! That works. Thanks for the quick implementation.
